lib/jsdom/browser/Window.js

```diff
@@ -503,6 +504,18 @@ function installOwnProperties(window, options) {
     screen: makeReplaceablePropertyDescriptor("screen", window),
     origin: makeReplaceablePropertyDescriptor("origin", window),
     event: makeReplaceablePropertyDescriptor("event", window),
+    TextEncoder: {
+      value: TextEncoder,
+      configurable: true,
+      enumerable: false,
+      writable: true
+    },
+    TextDecoder: {
+      value: TextDecoder,
+      configurable: true,
+      enumerable: false,
+      writable: true
+    },
```

Comment:
I used this because this is what I have in my browser when I run `Object.getOwnPropertyDescriptor(window, 'TextEncoder')`:

<img width="487" alt="image" src="https://github.com/user-attachments/assets/66b95bc7-26d8-4aff-b183-f2b67387e97d">

Reply:
I didn't use `makeReplaceablePropertyDescriptor` because it doesn't seem to be using a `set` & a `get`, but I can change that if needed

<img width="547" alt="image" src="https://github.com/user-attachments/assets/54d11f0f-6ff3-435b-8c30-555e2e885a86">